pynest/nest/tests/test_connect_arrays_mpi.py

```diff
@@ -57,7 +57,11 @@ class ConnectArraysMPICase(unittest.TestCase):
     multiple processes.
     """
     non_unique = np.array([1, 1, 3, 5, 4, 5, 9, 7, 2, 8], dtype=np.uint64)
-    comm = MPI.COMM_WORLD.Clone()
+
+    # We will enter the test class if HAVE_MPI4PY is false (though all tests are then skipped), so we need to only
+    # set the variable if we have mpi4py.
```

Comment:
I find "enter" a bit general here. Here's a suggestion that works better for me (in case I understand the mechanism correctly, anyway):
```suggestion
    # The test class is instantiated by the unittest framework regardless of the value of
    # HAVE_MPI4PY, even though all tests will be skipped in case it is False. In this
    # situation, we have to manually prevent calls to MPI in order to avoid errors during
    # the execution.
```